apache_mod_wsgi/README.md

```diff
@@ -0,0 +1,113 @@
+# Setting up and configuring Shibboleth SP with UMass web login 
+
+1. Getting SP config files.
+2. Downloading server prerequisites.
+3. Create/apply a ssl cert. 
+4. Configuring Shibboleth.
+5. Checking.
+
+
+
+## Getting SP config files
+
+The first step you want to do is to request SP authentication and configuration files.
+
+Go to https://webauth.umass.edu/admin and there you can create and manage pre-existing SP tokens.
+
+Select create new SP and follow the steps using the domain of your app. *For local use and testing, just use `localhost` as your domain.*
+
+## Downloading server prerequisites.
+
+For this project we are using Ubuntu as our server OS. 
+
+Enter these into your terminal in order. 
```

Comment:
Feel like having a colon instead of a period would make it clearer to indicate what commands should be written down.